awscli/customizations/eks/get_token.py

```diff
@@ -13,6 +13,8 @@
 import base64
```

Comment:
It would also be great if we can get a changelog for this feature? You can use the [`scripts/new-change`](https://github.com/aws/aws-cli/blob/develop/scripts/new-change) script to generate a changelog entry. For the type field, I'd recommend setting the type to `enhancement` and the category to `eks get-token`